onnxruntime/test/shared_lib/test_inference.cc

```diff
@@ -743,10 +743,19 @@ TEST(CApiTest, io_binding_cuda) {
   };
 
   Ort::SessionOptions session_options;
-  Ort::ThrowOnError(OrtSessionOptionsAppendExecutionProvider_CUDA(session_options, 0));
+  #ifdef USE_TENSORRT
+    Ort::ThrowOnError(OrtSessionOptionsAppendExecutionProvider_Tensorrt(session_options, 0));
+  #else
+    Ort::ThrowOnError(OrtSessionOptionsAppendExecutionProvider_CUDA(session_options, 0));
+  #endif
   Ort::Session session(*ort_env, MODEL_URI, session_options);
 
-  Ort::MemoryInfo info_cuda("Cuda", OrtAllocatorType::OrtArenaAllocator, 0, OrtMemTypeDefault);
+  #ifdef USE_CUDA
+    Ort::MemoryInfo info_cuda("Tensorrt", OrtAllocatorType::OrtArenaAllocator, 0, OrtMemTypeDefault);
```

Comment:
Is this reversed? 
Shouldn't it be info_cuda("Cuda", ... ) ?

Reply:
actually, i think it needs to be 
ifdef USE_TENSORRT  , rather than changing info_cuda
since both will be true when trt is enabled

Reply:
yes. I did #ifdef  USE_CUDA, then realized this as well, then fixed it, but missed the second place.

---
In reply to: [550707844](https://github.com/microsoft/onnxruntime/pull/6240#discussion_r550707844) [](ancestors = 550707844)